
Backport of PEP 526, syntax for variable annotations, to py2 lib2to3's grammar.
Applies to https://github.com/python/cpython/blob/2.7/Lib/lib2to3/Grammar.txt at
commit 829d4fbbf93a25424ec007808617d87deb63eec9.

diff --git a/pytype/tools/merge_pyi/Grammar.txt b/pytype/tools/merge_pyi/Grammar.txt
index 9be7c9f0..fc72945c 100644
--- a/pytype/tools/merge_pyi/Grammar.txt
+++ b/pytype/tools/merge_pyi/Grammar.txt
@@ -53,8 +53,9 @@ stmt: simple_stmt | compound_stmt
 simple_stmt: small_stmt (';' small_stmt)* [';'] NEWLINE
 small_stmt: (expr_stmt | print_stmt  | del_stmt | pass_stmt | flow_stmt |
              import_stmt | global_stmt | exec_stmt | assert_stmt)
-expr_stmt: testlist_star_expr (augassign (yield_expr|testlist) |
+expr_stmt: testlist_star_expr (annassign | augassign (yield_expr|testlist) |
                      ('=' (yield_expr|testlist_star_expr))*)
+annassign: ':' test ['=' test]
 testlist_star_expr: (test|star_expr) (',' (test|star_expr))* [',']
 augassign: ('+=' | '-=' | '*=' | '@=' | '/=' | '%=' | '&=' | '|=' | '^=' |
             '<<=' | '>>=' | '**=' | '//=')
